connectivity/check/action.go

```diff
@@ -123,6 +123,7 @@ func (a *Action) Destination() TestPeer {
 // This method is to be called from a Scenario implementation.
 func (a *Action) Run(f func(*Action)) {
 	a.Logf("[.] Action [%s]", a)
+	a.Progress()
```

Comment:
Can probably call `a.test.Progress()` here and remove the `Action.Progress()` layer of indirection. The intention is more for an Action to log progress against the Test, not against itself.

Reply:
Good point, changed as requested.